client/js/bundles/procedure/publicIndexMapAndList.js

```diff
@@ -14,9 +14,13 @@
 import { dpApi, prefixClass } from '@demos-europe/demosplan-ui'
 import DpSearchProcedureMap from '@DpJs/components/procedure/publicindex/DpSearchProcedureMap'
 import { initialize } from '@DpJs/InitVue'
+import RegisterFlyout from '@DpJs/components/user/RegisterFlyout'
 
 const stores = {}
-const components = { DpSearchProcedureMap }
+const components = {
+  DpSearchProcedureMap,
+  RegisterFlyout
```

Comment:
I'm wondering if this is another case for registering a component globally... like we already do it with [some other components](https://github.com/demos-europe/demosplan-core/blob/21da7fe683e43c2151fadd59ef6a2d06cbeb0b4e/client/js/InitVue.js#L64-L68) that are required basically on every page.

Reply:
I think adding this component globally is less troublesome than adding it on multiple entry points for the same result, hence added it globally here: https://github.com/demos-europe/demosplan-core/pull/2366/commits/e0fa4600232ad27e204250d7a239759864279120